packages/thumbprint-react/components/Tooltip/index.tsx

```diff
@@ -213,13 +248,26 @@ export default function Tooltip({
                                     // devices so it could easier to be toggled off.
                                     if (doesWindowSupportTouch()) {
                                         hide();
+                                        setHasOpenedByOnClick(false);
                                     }
                                 }}
                             >
                                 {/* We need to let the popper instance know when the contents of the tooltip change,
                                 so it can reposition itself.
                                 https://github.com/thumbtack/thumbprint-archive/issues/1033 */}
-                                <WhenChildrenChange do={scheduleUpdate}>{text}</WhenChildrenChange>
+                                <WhenChildrenChange do={scheduleUpdate}>
+                                    <div
+                                        onFocus={(): void => {
+                                            if (closeTimeout) {
+                                                window.clearTimeout(closeTimeout);
+                                            }
+                                        }}
+                                        onBlur={hide}
+                                        className="pre-line"
```

Comment:
We don't use Atomic classes in Thumbprint React due to some versioning issues it would introduce. You'll need to add to the Sass file.

Reply:
Good to know! Updated!